packages/assets-controllers/src/TokenRatesController.test.ts

```diff
@@ -157,7 +157,7 @@ describe('TokenRatesController', () => {
         );
 
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        tokenStateChangeListener!({
+        await tokenStateChangeListener!({
```

Comment:
Any suggestions? It's still a test-only change, and was motivated by the ESLint warnings. I'm not sure what the PR title should be conveying beyond that.

It doesn't really impact the flow of the tests because the tests missing an `await` were all stubbing out the only async step of the exchange rate update process. That's why none of the test assertions required updating.